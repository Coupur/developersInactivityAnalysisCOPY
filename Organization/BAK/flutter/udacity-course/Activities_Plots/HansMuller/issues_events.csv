id,date,event,creator_login
1529686164,2018-03-19 22:30:08,mentioned,HansMuller
1529686165,2018-03-19 22:30:08,subscribed,HansMuller
1531554419,2018-03-20 17:40:41,mentioned,HansMuller
1531554421,2018-03-20 17:40:41,subscribed,HansMuller
