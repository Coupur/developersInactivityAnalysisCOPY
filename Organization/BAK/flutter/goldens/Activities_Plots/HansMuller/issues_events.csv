id,date,event,creator_login
1663877916,2018-06-05 14:36:24,referenced,HansMuller
1663877919,2018-06-05 14:36:24,merged,HansMuller
1663877927,2018-06-05 14:36:24,closed,HansMuller
1662455737,2018-06-04 22:53:14,referenced,HansMuller
1662455740,2018-06-04 22:53:14,merged,HansMuller
1662455741,2018-06-04 22:53:14,closed,HansMuller
